virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
 
-The Scheduling, Scale, and Performance (SSP) Operator integrates {VirtProductName} with {pipelines-shortname}. The SSP Operator includes tasks and example pipelines that allow you to:
+The {pipelines-shortname} tasks and example pipeline allow you to do the following:
 
-* Create and manage virtual machines (VMs), persistent volume claims (PVCs), and data volumes
+* Create and manage virtual machines (VMs), persistent volume claims (PVCs), data volumes and data sources
 * Run commands in VMs
 * Manipulate disk images with `libguestfs` tools
 
+The tasks are located in link:https://artifacthub.io/packages/search?repo=redhat-tekton-tasks&sort=relevance&page=1[the task catalog].
+
+The example Windows pipeline is located in link:https://artifacthub.io/packages/tekton-pipeline/redhat-pipelines/windows-efi-installer[the pipeline catalog].
```

Comment:
Link points to an article called "Windows EFI Installer Pipeline". Consider adding (ArtifactHub) after the article name so customers know they need to leave the OCP docs.